src/Tracker/onfidoTracker.tsx

```diff
@@ -51,6 +52,32 @@ const source_metadata = {
 
 const stepsArrToString = () => steps?.map((step) => step['type']).join()
 
+const removeKeys = [
+  'title',
+  'descriptions',
+  'nextButton',
+  'message',
+  'submessage',
+]
+const cleanStepsForConfig = (steps: StepConfig[]): StepConfig[] =>
```

Comment:
I think it would be better to pick only the configuration we want to include rather than removing those we don't want. Otherwise, any new step configuration will automatically be tracked and we would easily lose track of what we're sending.